doc/development_guide/contributor_guide/governance.rst

```diff
@@ -0,0 +1,95 @@
+============
+ Governance
+============
+
+How to become a triager ?
+-------------------------
+
+- Contribute for more than 3 releases consistently.
+- Do not be too opinionated, follow the code of conduct without requiring emotional
+  works from the maintainers. It does not mean that disagreements are impossible,
+  only that arguments should stay technical and reasonable so the conversation
+  is civil and productive.
+- Have a maintainer suggest that you become triager, without you asking
+- Get unanimous approval or neutral agreement from current maintainers.
+
+How to become a maintainer ?
+----------------------------
+
+- Contribute two big code merge requests over multiple releases (for example
+  one checker in 2.13 and the following bug after release and one complicated
+  bug fixes in 2.14). Otherwise contributing for more than 3 releases consistently
+  with great technical and interpersonal skills.
+- Take ownership of a part of the code that is not maintained well at the moment
+  or that you contributed personally (if we feel we can't merge something without
+  your review, you're going to be able to merge those yourself soon).
+- Have an admin suggest that you become maintainer, without you asking
+- Get unanimous approval or neutral agreement from current maintainers.
```

Comment:
Maybe it's self explanatory so not sure it should be added:
> Have been a triager for x amount of time / multiple months

If it's added, something similar could be added to the admin section as well.

Reply:
[0c634e3](https://github.com/PyCQA/pylint/pull/8329/commits/0c634e3f3730eb8e1f2ac9592b2ea5b3173cf90a)